agent/bench-scripts/pbench-trafficgen

```diff
@@ -856,39 +848,21 @@ if [ "${postprocess_only}" == "n" ]; then
 	# Ensure all specified devices are properly configured via dpdk.
 	configure_devices "${devices}"
 
-	if [ -d $trafficgen_dir ]; then
-		if [ $skip_git_pull == "n" ]; then
-			if pushd $trafficgen_dir >/dev/null && git pull && git checkout master; then
-				echo "trafficgen is up to date"
-				trafficgen_version=$(git log --max-count=1 --pretty=%h)
-				popd >/dev/null
-			else
-				error_log "could not pull latest trafficgen"
-				exit 1
-			fi
-		fi
+	if [[ ! -d ${trafficgen_dir} ]]; then
+		error_log "Installation of trafficgen could not be found at: ${trafficgen_dir}"
+		exit 1
+	fi
+	if [[ -f ${trafficgen_dir}/VERSION ]]; then
+		trafficgen_version=$(cat ${trafficgen_dir}/VERSION)
 	else
-		if pushd /opt >/dev/null && git clone https://github.com/atheurer/trafficgen.git; then
-			trafficgen_version=$(git log --max-count=1 --pretty=%h)
-			echo "trafficgen cloned"
-			popd >/dev/null
-		else
-			error_log "could not clone latest trafficgen"
-			exit 1
-		fi
+		trafficgen_version="unknown"
```

Comment:
`trafficgen_version` is initialized to this at line 100...do we need this assignment here?